update security consideration (#68)
ietf-wg-alto/draft-ietf-alto-oam-yang: others.md

restrict access for particular users to a pre-configured subset of all
available protocol operations and content.

There are a number of data nodes defined in these two YANG modules that are
writable/creatable/deletable (i.e., config true, which is the default).
These data nodes may be considered sensitive or vulnerable in some network environments.
Write operations (e.g., edit-config) to these data nodes without proper protection
can have a negative effect on network operations. These are the subtrees and data
nodes in "ietf-alto" YANG module and their sensitivity/vulnerability:

>   /alto:alto/alto:alto-client/alto:server-discovery:

Reviewer: ```suggestion
  /alto/alto-client/server-discovery:
```

>   : This subtree specifies a set of parameters for an ALTO client to discover ALTO server.

Reviewer: ```suggestion
  : This subtree specifies a set of parameters for an ALTO client to discover ALTO servers.
```

  Unauthorized access to it could cause intruders to modify the ALTO discovery

>   parameters(e.g., dns-server) so as to expose an ALTO client to fake ALTO servers.

Reviewer: ```suggestion
  parameters (e.g., "dns-server") in order to expose an ALTO client to fake ALTO servers. Likewise, this data node can be manipulated to prevent an ALTO client from discovering a reachable ALTO server.
```

>   /alto:alto/alto:alto-server/logging-system:

Reviewer: ```suggestion
  /alto/alto-server/logging-system:
```

  : This subtree provides configuration to select a logging system to capture log
  messages generated by an ALTO server. Unauthorized read access of this node

>   can allow intruders to access logging information, which could be used to craft

Reviewer: This item is about readable concerns, while the items in this list should focus on "writable/creatable/deletable". Please move this one to be positioned right after: 

> Some of the readable data nodes in this YANG module may be considered sensitive or vulnerable in some network environments. It is thus important to control read access (e.g., via get, get-config, or notification) to these data nodes. These are the subtrees and data nodes and their sensitivity/vulnerability:

Author: Sure, finished.

  an attack the server.

>   /alto:alto/alto:alto-server/auth-client:

Reviewer: ```suggestion
  /alto/alto-server/auth-client:
```

  : This list specifies all the authenticated ALTO clients on an ALTO server.
  Unauthorized write access to this list can allow intruders to modify the entries
  so as to add a client that have not been authenticated yet or delete a client

>   that has already been authenticated.

Reviewer: ```suggestion
  that has already been authenticated. Likewise, this data node can be manipulated to prevent access of legitimate ALTO clients.
```

  /alto:alto/alto:alto-server/role:
  : This list specifies roles which authenticated ALTO clients assigned to for
  access control. Unauthorized write access to this list can allow intruders to
  modify the entries so as to permit access that should not be permitted, or deny
  access that should be permitted.

>   /alto:alto/alto:alto-server/data-source/feed-interval:

Reviewer: ```suggestion
  /alto/alto-server/data-source/feed-interval:
```
